CMakeLists.txt

```diff
@@ -370,10 +370,15 @@ INCLUDE_DIRECTORIES (${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR} ${CMAKE_SOU
 
 MESSAGE (STATUS "Looking for modules in ${CMAKE_MODULE_PATH}")
 
-set (CMAKE_C_FLAGS     "${CMAKE_C_FLAGS} -Wall")
-set (CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS} -O3")
-set (CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS} -g -O3")
-set (CMAKE_C_FLAGS_DEBUG     "${CMAKE_C_FLAGS} -g3 -O0")
+IF (NOT MSVC)
+  set (CMAKE_C_FLAGS                "${CMAKE_C_FLAGS} -Wall")
+  set (CMAKE_C_FLAGS_RELEASE        "${CMAKE_C_FLAGS} -O3")
+  set (CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS} -g -O3")
+  set (CMAKE_C_FLAGS_DEBUG          "${CMAKE_C_FLAGS} -g3 -O0")
+ELSE ()
+  set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /wd4244 /wd4267")
+  add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+ENDIF ()
```

Comment:
Can you please flip this to be `IF (MSVC) ... ELSE () ... ENDIF ()`?

Reply:
done